profile/src/main/java/org/apache/karaf/profile/assembly/AssemblyDeployCallback.java

```diff
@@ -154,45 +154,56 @@ public void installConfigs(org.apache.karaf.features.Feature feature) throws IOE
         // Install
         Downloader downloader = manager.createDownloader();
         for (Config config : ((Feature) feature).getConfig()) {
+            Path configFile = etcDirectory.resolve(config.getName() + ".cfg");
+            if (Files.exists(configFile) && !config.isAppend()) {
```

Comment:
Maybe, we can also introduce `config.isOverride()` to completely replace a existing configuration with the one contained in the feature (as we have for `<configFile/>`. Tought ?

Reply:
it'd require XSD version change. adding explicit `override` would be probably confusing - at runtime, when feature resolver installs (or even refreshes) a feature, it would have **not** to overwrite it anyway

Reply:
OK fair enough. Np